.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Description
+[PLEASE REMOVE] Describe the goal of this PR and any add any relevant context.
+
+### Test plan
+[PLEASE REMOVE] How did you test this PR? Please write down any manual commands you used and note down tests that you have written.
+
+### Issue(s) fixed
+[PLEASE REMOVE] List all issues, separate by returns, that this PR addresses.
```

Comment:
For this part, we can simply use this feature from GH https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword